tools/add_triage_signature.py

```diff
@@ -345,6 +345,33 @@ def _get_hostname(host):
         inventory = json.loads(host["inventory"])
         return inventory["hostname"]
 
+    @staticmethod
+    def _generate_hosts_summary(hosts):
+        host_summary = []
+        for host in hosts:
+            # Add host status summary
+            if host["role"] not in ["master", "boostrap"]:
+                continue
+            if host["progress"] == "Done" or host["status"] != "error":
+                continue
+
+            host_comment = {
+                "Waiting for bootkube": "Bootkube.service never started",
```

Comment:
It means bootkube [didn't finish](https://github.com/openshift/assisted-installer/blob/c13498b0e12bbe4ca3c1eea5e677b75712bba78b/src/installer/installer.go#L582), not that it didn't start